src/server/controllers/getCount.js

```diff
@@ -1,13 +1,38 @@
+const { Op } = require('sequelize');
+
 const participant = require('../database/models/participant');
 const course = require('../database/models/course');
 const worker = require('../database/models/worker');
 
+
 exports.get = async (req, res) => {
   try {
     const countParticipant = await participant.findAndCountAll();
     const countCourse = await course.findAndCountAll();
     const countWorker = await worker.findAndCountAll();
-    res.send({ countParticipant, countCourse, countWorker });
+    const notEmployed = 'not employed yet';
+    const employed = 'employed';
+    const countEmployedParticipant = await participant.findAndCountAll({
+      where: {
+        employment_outcomes: {
+          [Op.like]: `${employed}`,
+        },
+      },
+    });
+    const countNotEmployedParticipant = await participant.findAndCountAll({
```

Comment:
If the only options for employment_outcomes are employed or not employed, then you could just calculate countNotEmployed as “countParticipants - countEmployed”. Then it’s one less database query, which means slightly better performance. Not a required change though, just an idea for optimization 😀